### ISSUE TITLE
Heya did you have a link to the weights and the decode map ?

### ISSUE THREAD

**Issue by kpayne1** [jc639/pytorch-handwritingCTC#1]
Heya awesome work rebuilding this in PyTorch, I'm curious as to the method you used to adjust the iam handwriting dataset to work with the custom dataset class are did you work with it as it came,did you have a link to the weights and the decode map?
i attached the file you would have context as to what i mean 
[lines.txt](https://github.com/jc639/pytorch-handwritingCTC/files/4103119/lines.txt)

**Comment by jc639** [comment]
Hey, I didn't do much else other than convert the text file to CSV. The file needs to match the format here: [dataset format](https://github.com/jc639/pytorch-handwritingCTC#dataset-format-and-dataloaders)

I did, however, deslant the images using the dataset class and resave the deslanted versions as a preprocessing step. I have added a jupyter notebook to show how I did that [here](https://github.com/jc639/pytorch-handwritingCTC/blob/master/Deslant%20Dataset.ipynb). This might give a bit more insight into the dataset class.

I have made the weights available [here](https://drive.google.com/drive/folders/1hwPI13LCVF52WQboSfr_4jYsGIGV1tle?usp=sharing). The word model requires input size of 64x256, and the line model 64x800, so set the output_size parameter of the Rescale transform to `output_size=(64, 800)` for the line model for instance.

You can train from 'scratch' as well to see if other regularisation levels etc make a difference. As there is transfer learning from the imagenet weights for the conv part it only took an hour or so to train. One thing to note, I have realised that I freeze the conv part initially but you probably don't want to freeze the batch norm layers when you start training - this could be adjusted. 

I hope the code is reproducible, but I reckon it could be better so apologies about that. You will need pytorch 1.1 or higher, pandas, numpy, scikit-image and opencv. If you give it a go, and get stuck or whatever - let me know and I will see if I can help out.

**Comment by kpayne1** [comment]
thanks, man i actually was able to run inference on a few images, gonna train with the pre-trained model on my dataset
